### PR TITLE
update Authors

### PR DESCRIPTION
**Pull request by RituRajSingh878**
#### Brief description of what is fixed or changed
update authors list

#### Release Notes

 NO ENTRY

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -757,3 +757,35 @@ Ayushman Koul <ayushmankoul4570@gmail.com>
...
+SirJohnFranklin <sirjfu@googlemail.com>
+Anthony Sottile <asottile@umich.edu>
+ViacheslavP <public.viacheslav@gmail.com>
+Safiya03 <safiyanesar@gmail.com>
+Alexander Dunlap <alexander.dunlap@gmail.com>
+Rohit Sharma <31184621+rohitx007@users.noreply.github.com>
+Jonathan Warner <warnerjon12@gmail.com>
+Mohitbalwani26 <44258119+Mohitbalwani26@users.noreply.github.com>
+Marduk Bolaños <mardukbp@mac.com>
+mardukbp <mardukbp@mac.com>
```
Comment (oscargus) [inline AUTHORS]: Seems to be the same person.

Comment (RituRajSingh878) [reply AUTHORS]: thanks for pointing out.

**Comment by oscargus** [comment]
Is this based on the very latest master? (I would expect my name to show up, but have no idea how it works.)

**Comment by RituRajSingh878** [comment]
> Is this based on the very latest master? (I would expect my name to show up, but have no idea how it works.)

yes, it is based on the latest master.

**Comment by asmeurer** [comment]
Please also update .mailmap. Make sure you are using the `bin/mailmap_update.py` and `bin/authors_update.py`. Every author name should be normalized and deduplicated with .mailmap.

File: AUTHORS [hunk 2]
```diff
@@ -757,3 +757,33 @@ Ayushman Koul <ayushmankoul4570@gmail.com>
...
+SirJohnFranklin <sirjfu@googlemail.com>
+Anthony Sottile <asottile@umich.edu>
+ViacheslavP <public.viacheslav@gmail.com>
+Safiya03 <safiyanesar@gmail.com>
+Alexander Dunlap <alexander.dunlap@gmail.com>
+Rohit Sharma <31184621+rohitx007@users.noreply.github.com>
+Jonathan Warner <warnerjon12@gmail.com>
+Mohitbalwani26 <44258119+Mohitbalwani26@users.noreply.github.com>
+Marduk Bolaños <mardukbp@mac.com>
+amsuhane <“ayushsuhane99@iitkgp.ac.in”>
```
Comment (asmeurer) [inline AUTHORS]: Remove the Unicode quotes with .mailmap.

**Comment by asmeurer** [comment]
You may also need to ping people here to see which name or email they prefer to use.

**Comment by RituRajSingh878** [comment]
@apalmer1377  and @mardukbp 
you people have these names showing in Authors list, so which one of the names you people want to be in the Author list.

Austin Palmer <ap4000@nyu.edu>
	austin <ap4000@nyu.edu>

	Marduk Bolaños <mardukbp@mac.com>
	mardukbp <mardukbp@mac.com>

**Comment by mardukbp** [comment]
Ups! Sorry for that. The first one please.

**Comment by apalmer1377** [comment]
The first one, please!

**Comment by RituRajSingh878** [comment]
I don't know, Why sage tests are failing?

**Comment by asmeurer** [comment]
It's an unrelated issue https://github.com/sympy/sympy/issues/15802.

File: AUTHORS [hunk 3]
```diff
@@ -757,3 +757,35 @@ Ayushman Koul <ayushmankoul4570@gmail.com>
...
+Shubham Kumar Jha <skjha832@gmail.com>
+rationa-kunal <kunalgk1999@gmail.com>
+Animesh Sinha <animeshsinha1309@gmail.com>
+Gaurang Tandon <1gaurangtandon@gmail.com>
+Matthew Craven <clyring@users.noreply.github.com>
+Daniel Ingram <ingramds@appstate.edu>
+Jogi Miglani <jmig5776@gmail.com>
+Takumasa Nakamura <n.takumasa@gmail.com>
+RituRajSingh878 <RituRajSingh878@gmail.com>
+Rajiv Ranjan Singh <42106787+iamrajiv@users.noreply.github.com>
```
Comment (asmeurer) [inline AUTHORS]: These two are duplicates. Are you using the authors script to generate this file?

File: .mailmap [hunk 4]
```diff
@@ -397,3 +397,7 @@ Gagandeep Singh <singh.23@iitj.ac.in> MathLover <36567889+czgdp1807@users.norepl
 S.Y. Lee <sylee957@gmail.com> sylee957 <sylee957@gmail.com>
 Yatna Verma <yatnavermaa@gmail.com> yatna <yatnavermaa@gmail.com>
 Jared Lumpe <mjlumpe@gmail.com> Michael Jared Lumpe <mjlumpe@gmail.com>
+Austin Palmer <ap4000@nyu.edu> austin <ap4000@nyu.edu>
+Marduk Bolaños <mardukbp@mac.com> mardukbp <mardukbp@mac.com>
+amsuhane <ayushsuhane99@iitkgp.ac.in> amsuhane <“ayushsuhane99@iitkgp.ac.in”>
+RituRajSingh878 <RituRajSingh878@gmail.com> Ritu Raj Singh <37741324+RituRajSingh878@users.noreply.github.com>
```
Comment (asmeurer) [inline .mailmap]: Is `RituRajSingh878` how you want it to appear in the AUTHORS? I would suggest

```
Ritu Raj Singh <RituRajSingh878@gmail.com> RituRajSingh878 <RituRajSingh878@gmail.com>
Ritu Raj Singh <RituRajSingh878@gmail.com> Ritu Raj Singh <37741324+RituRajSingh878@users.noreply.github.com>
``` 

But it's up to you which name/email combo you want to appear.

File: AUTHORS [hunk 5]
```diff
@@ -757,3 +757,35 @@ Ayushman Koul <ayushmankoul4570@gmail.com>
...
+Alexander Dunlap <alexander.dunlap@gmail.com>
+Rohit Sharma <31184621+rohitx007@users.noreply.github.com>
+Jonathan Warner <warnerjon12@gmail.com>
+Mohitbalwani26 <44258119+Mohitbalwani26@users.noreply.github.com>
+Marduk Bolaños <mardukbp@mac.com>
+amsuhane <ayushsuhane99@iitkgp.ac.in>
+Matthias Geier <Matthias.Geier@gmail.com>
+klaasvanaarsen <44929042+klaasvanaarsen@users.noreply.github.com>
+Shubham Kumar Jha <skjha832@gmail.com>
+rationa-kunal <kunalgk1999@gmail.com>
```
Comment (asmeurer) [inline AUTHORS]: This is probably a duplicate with the `kunal` at the bottom of the file.

Comment (RituRajSingh878) [reply AUTHORS]: I am pinging him to know, which one name/email combination he wants?

File: .mailmap (hunk 4 above)
Comment (RituRajSingh878) [reply .mailmap]: thanks for the suggestion.

File: AUTHORS (hunk 3 above)
Comment (RituRajSingh878) [reply AUTHORS]: yes,  I am using authors script to generate this file.

Comment (RituRajSingh878) [reply AUTHORS]: > These two are duplicates.

which one?

Comment (asmeurer) [reply AUTHORS]: Oh sorry, I thought Rajiv Ranjan Singh and you were the same person (you have very similar names).

**Comment by RituRajSingh878** [comment]
@asmeurer can you check it?

**Comment by asmeurer** [comment]
It looks good to me. We need to get the Sage situation sorted before we can merge anything.

**Comment by asmeurer** [comment]
It looks like it isn't possible to fix the Travis tests by restarting them, so you will need to push another commit here to get the build passing. If you have no other changes to make, you can just merge the latest master and push that.

**Comment by RituRajSingh878** [comment]
@asmeurer now it can merge into master.

**Pull request merged**